commit 1ce99798fbb4b2086cbcce67907191ae36aadbb8 (refs/changes/61/3761/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-04-03T10:46:11-07:00 (1 year, 6 months ago)
    
    TRITON-288 sdcnode updates for March 2018 node.js security releases
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/Makefile b/Makefile
index 2af1711..e1823f8 100644
--- a/Makefile
+++ b/Makefile
@@ -50,7 +50,7 @@ CLEAN_FILES	+= node_modules cscope.files
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v4.8.7
+NODE_PREBUILT_VERSION=v4.9.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG=zone
